software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeSshDriver.java

```diff
@@ -95,7 +95,8 @@ public void install() {
         OsDetails osDetails = getMachine().getMachineDetails().getOsDetails();
         List<String> commands = Lists.newLinkedList();
         if (osDetails.isLinux()) {
-            commands.addAll(installLinux(getExpandedInstallDir()));
+//            commands.addAll(installLinux(getExpandedInstallDir()));
```

Comment:
We use the scripts provided by package cloud.
It has two bash scripts for debian based and rpm based distros which handle the resolution for choosing a concrete package.
https://packagecloud.io/basho/riak/install